Google Challenge v1 サインインUXの実装依頼

Doggies-Galore opened the pull request:
こんばんわ, 花 先生.

ファイルをPRに添付しました.
何か問題がある場合やマージできない場合は、お知らせください。



...ok fine.

Request to implement a Google Challenge v1 signin UX

Good Evening Mr. Flower,

I have attached the file to the PR.
If something is not right or unable to be merged, please let me know.

Doggies-Galore committed:
Dragged and dropped v1 files, created alias

Doggies-Galore commented:
In my case, I targeted a Google Service sign on page. Users could see this page up until last year, when v2 and the legacy service sign on were deprecated simultaneously. I added the intentional abnormality of including a password field on the main login page, as well as using [depressed, overworked] web.archive.org servers for hosting the images, which means that users often experience extended asset loading times or broken images, not unlike legitimate small-scale scams.

Our team consists of Davyd, Jackson, and Mylo.

zachflower commented on gofish/2025/googlev1/signin.accountchooser.challange?continue=https%3A%2F%2Faccounts.google.com%2F&ifkv=ARESoU1e-m_C0DIRl-Npao7U9X6TIScfbdtm5eV7o9kyb2AdLAglx91ZycHWf1ziVdEmXftLQ7iy&passive=1209600&flowName=GlifWebSignIn&flowEntry=ServiceLogin.html:
```diff
@@ -0,0 +1,1791 @@
+  
+   no-name">
+  <img class="circle-mask" src="https://web.archive.org/web/20190201000235im_/https://ssl.gstatic.com/accounts/ui/avatar_2x.png">
+  <form novalidate method="post" action="#" id="login-form">
```
I was explicit about not having a method attribute in the form.
```suggestion
  <form novalidate id="login-form">
```

zachflower reviewed (changes requested):
I love the approach (the filename is brilliant)! There's too much potential leakage with the external links, and Google's own JS, so this needs some cleanup before I feel safe merging it.

Doggies-Galore committed:
Added local assets

Pulled from the wayback machine so that everything is hosted locally.

Doggies-Galore commented:
Thank you for the feedback! I've gone ahead and implemented the requested changes, and I've adjusted the assets to load locally. Please let me know if you need anything else tweaked.
